Clients/Xamarin.Interactive.Client.Windows/Views/InspectTreeView.xaml.cs

```diff
@@ -127,85 +150,107 @@ void OnLoaded (object sender, RoutedEventArgs e)
             DependencyProperty.Register (
                 nameof (ShowHidden),
                 typeof (bool),
-                typeof (VisualRepView),
+                typeof (InspectTreeView),
                 new PropertyMetadata (
                     false,
                     new PropertyChangedCallback (ShowHiddenValueChanged)));
 
         static void ShowHiddenValueChanged (DependencyObject dependencyObject, DependencyPropertyChangedEventArgs eventArgs)
         {
             var value = (bool)eventArgs.NewValue;
-            var view = dependencyObject as VisualRepView;
-            var state = new TreeState (view.DisplayMode, value);
-            view.currentViewNode.Rebuild (state);
+            var view = dependencyObject as InspectTreeView;
+            var state = new InspectTreeState (view.DisplayMode, value);
+            //view.representedNode.Rebuild (state);
```

Comment:
Might as well remove this?

Reply:
This was actually a bug.  Clean up and fix incoming.